saxml/tools/docker/Dockerfile.admin

```diff
@@ -42,18 +42,21 @@ ENV SAX_CELL=/sax/test
 # Use "10000" as default port for admin server gRPC endpoint.
 ENV PORT=10000
 
+ENV SAX_ROOT=gs://${GSBUCKET}/sax-root
+ENV FS_ROOT=gs://${GSBUCKET}/sax-fs-root
```

Comment:
I think this will make ${GSBUCKET} getting substituted at the docker build time. I am not familiar with docker cmds, but perhaps try to add and env for GSBUCKET? something like

ENV GSBUCKET=""

Reply:
Did not work, but adding an if statement inside the bash script inside the dockerfile did